### PR TITLE
Struct, methods & interface task

### PR DESCRIPTION
**Pull request by JuanCalcagno416**

### CODE REVIEW

File: structs-methods-interfaces/shapes_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		{shape: Rectangle{Width: 12, Height: 6}, want: 72.0},
+		{shape: Circle{Radius: 10}, want: 314.1592653589793},
+		{shape: Triangle{Base: 12, Height: 6}, want: 36.0},
+	}
+
+	for _, c := range areaCases {
+		checkArea(t, c.shape, c.want)
+	}
+
+}
```
Comment (rjar2020) [inline structs-methods-interfaces/shapes_test.go:51]: Where are the examples and benchmarks? :-D

**Comment by rjar2020** [comment]
Why won't you create an (GitHub) Action to run the tests every time you do a PR and/or merge code to master? Just to have some fun.

File: structs-methods-interfaces/shapes_test.go (hunk 1 above)
Comment (JuanCalcagno416) [reply structs-methods-interfaces/shapes_test.go:51]: I know. Will add them today :)

**Comment by JuanCalcagno416** [comment]
> Why won't you create an (GitHub) Action to run the tests every time you do a PR and/or merge code to master? Just to have some fun.

Will do it later. Just a PR like you did for the refactoring test or repo

**Review by rjar2020 — APPROVED** [review]
LGTM!

**Pull request merged**